docs/content/guides/author-apps/networking/howto-gateways/index.md

```diff
@@ -16,9 +16,9 @@ This guide will walk you through how to setup a gateway for routing internet tra
 - [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-radius-bicep-vs-code-extension" >}})
 - [Radius environment]({{< ref "installation#step-3-initialize-the-radius-control-plane-and-the-radius-environment" >}})
 
-## Step 1: Define a container
+## Step 1: View the container
```

Comment:
hmm without explicitly telling the user to run `rad init` and select "yes" to scaffold an application I think "Define a container" works better, along with "in a new file".

Reply:
The pre-requisite step Radius environment does cover the step to select yes and set up the application. Either we go with that or make env creation a separate step here

Reply:
closing as we decided to make how-tos standalone and not operate with the assumption of user scaffolding an application via rad init